vault/resource_approle_auth_backend_role_secret_id_test.go

```diff
@@ -37,6 +37,18 @@ func TestAccAppRoleAuthBackendRoleSecretID_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet(secretIDResource, "accessor"),
 				),
 			},
+			{
```

Comment:
Confirmed that this test will fail on main and successfully apply and pass with the fixes in this PR